IonicPortals/IonicPortals/PortalsPlugin.swift

```diff
@@ -12,7 +12,7 @@ public class PortalsPlugin: CAPPlugin {
     
     // MARK: Methods used by Cap Plugin
 
-    @objc func publishNative(_ call: CAPPluginCall) {
```

Comment:
These methods are specific to the capacitor plugin interface, are you crafting a `CAPPluginCall` on the NativeScript side? If not we'll have to make a separate `@objc` interface.

Reply:
Ah you're right, it's the static methods I meant to annotate, will update the PR once I figure a way around ` function types cannot be represented in Objective-C unless their parameters and returns can be`

Reply:
It would require either making `SubscriptionResult` a class that inherits from `NSObject`, but I'm not 100% sure I want to go that route. We have our sprint planning tomorrow and I was going to bring this very subject up